ansible/roles/service/hippo_authoring/defaults/main.yml

```diff
@@ -3,17 +3,20 @@
 hippo_authoring:
   user: hippo_authoring
   domain: hippo_authoring
-  # how to deploy / download Hippo
+  # how to download Hippo
   # - internal
   # - local
   # - s3
-  deploy_type: ~
+  download:
+    type: ~
```

Comment:
Suggest renaming to `source_type` for clarity, since I'm guessing possible values are local or s3?

Reply:
good suggestion ... :)